src/request/request.ts

```diff
@@ -56,10 +56,19 @@ export class Request implements IRequest {
     return undefined
   }
 
-  public header(key: string): string {
+  public header(key: string): string | undefined {
     if (typeof this.event.headers[key] !== 'undefined') {
       return this.event.headers[key]
     }
+    
+    return undefined
```

Comment:
cant we just return `this.event.headers[key]`? it will be undefined if its not set right?